kover/kover-cli/src/main/kotlin/kotlinx/kover/cli/commands/OfflineInstrumentCommand.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2000-2023 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kotlinx.kover.cli.commands
+
+import com.intellij.rt.coverage.instrument.Instrumentator
+import com.intellij.rt.coverage.report.data.Filters
+import kotlinx.kover.cli.util.asPatterns
+import org.kohsuke.args4j.Argument
+import org.kohsuke.args4j.Option
+import java.io.File
+import java.io.PrintWriter
+
+
+internal class OfflineInstrumentCommand : Command {
+    // hint: MutableList used to remove variance, args4j accept java.util.List<File> but not java.util.List<? extends File>
+    @Argument(usage = "list of the compiled class-files roots", metaVar = "<class-file-path>")
+    private var roots: MutableList<File> = ArrayList()
+
+    @Option(name = "--dest", usage = "path to write instrumented Java classes to", metaVar = "<dir>", required = true)
+    private var outputDir: File? = null
+
+    @Option(name = "--hits", usage = "path to write instrumented Java classes to")
```

Comment:
Here usage string is unrelated. Should be `a flag to enable line hits counting`?